GiftGivingGenerator.API/Controllers/DrawingResultsController.cs

```diff
@@ -24,15 +33,42 @@ public ActionResult GenerateDrawingResults([FromRoute] Guid eventId)
 		var @event = _eventRepository.Get(eventId);
 		var numberOfTries = @event.DrawResultsAndNumberTries();
 		Log.Information($"For event {eventId} I trie {numberOfTries} times to draw result.");
-
 		_eventRepository.Update(@event);
+		
+		var organizer = _personRepository.Get(@event.OrganizerId);
```

Comment:
get all required data from repository using projection

Reply:
Can be it in this way? - see commit